src/index.ts

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+import { Command } from "commander";
+
+import { init } from "@/commands/init";
+import { getPackageInfo } from "@/utils/get-package-info";
+import { logger } from "@/utils/logger";
+
+process.on("SIGINT", () => process.exit(0));
+process.on("SIGTERM", () => process.exit(0));
+
+async function main() {
+  const packageInfo = await getPackageInfo();
+
+  const program = new Command()
+    .name("atomic-ui")
```

Comment:
https://github.com/senantix/atomicui/pull/1/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R12

I updated the string name here - this is just for the help prompt

This will be the name of the script so it would be
`npx atomic init`